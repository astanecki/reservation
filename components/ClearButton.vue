<template>
  <button class="clear-button">Clear</button>
</template>
<style lang="scss" scoped>
.clear-button {
  align-self: flex-end;
  border: 1px solid #c6c6c6;
  border-radius: 5px;
  background-color: indianred;
  color: white;
  padding: 5px;
  font-size: 15px;
  width: 50px;

  cursor: pointer;
}
</style>
