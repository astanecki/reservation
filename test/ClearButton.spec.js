import { mount } from "@vue/test-utils";

import ClearButton from "~/components/ClearButton";

describe("ClearButton", () => {
  const createWrapper = () => {
    return mount(ClearButton);
  };

  it("is a Vue instance", () => {
    const wrapper = createWrapper();

    expect(wrapper.vm).toBeTruthy();
  });

  it(`should match initial snapshot`, () => {
    const wrapper = createWrapper();

    expect(wrapper.vm.$el).toMatchSnapshot();
  });
});
