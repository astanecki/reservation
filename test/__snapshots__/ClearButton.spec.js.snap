// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClearButton should match initial snapshot 1`] = `
<button
  class="clear-button"
>
  Clear
</button>
`;
